app/views/devise/passwords/edit.html.slim

```diff
@@ -14,6 +14,9 @@
         = f.input :reset_password_token, as: :hidden
         = f.full_error :reset_password_token
         = f.input :password, label: 'New password', required: true, autofocus: true
+        = render 'devise/shared/password_strength'
         = f.input :password_confirmation, label: 'Confirm your new password', required: true
         = f.button :submit, 'Change my password'
       = render 'devise/shared/links'
+
+== javascript_include_tag 'misc/pw-strength'
```

Comment:
This will not work in production because our CSP configuration does not allow inline JS. If we only want to include the JS on this page, why not use a CSS class based on the controller and action, and have the JS run if that class is present?
For example, we would update the `body` class in `application.html.slim` to this:
```
body class="#{Rails.env}-env site #{controller_name}-#{action_name}"
```
and then move the JS to a separate file and have it look for the PasswordsController edit action to determine whether or not to run this code.

See this article for more details about organizing JS in a Rails app: http://brandonhilkert.com/blog/organizing-javascript-in-rails-application-with-turbolinks/